README.md

```diff
@@ -26,220 +35,182 @@ required for testing citus.
 * [Advanced fab Usage](#advanced-fab)
   * [Using Multiple Citus Installations](#multiple-installs)
 
-# <a name="getting-started"></a> Getting Started
+## <a name="azure"></a>Azure
+
+## <a name="azure-getting-started"></a> Getting Started
 
 You can find more information about every step below in other categories. This list of commands show how to get started quickly. Please see other items below to understand details and solve any problems you face.
 
-## <a name="basic-cluster-setup"></a> Basic Cluster Setup
+You should have `az cli` in your local to continue.
 
-On your local machine:
-```bash
+You should use `ssh-agent` to add your ssh keys, which will be used for downloading the enterprise repository. Note that your keys are kept only in memory, therefore this is a secure step.
 
-# Add your EC2 keypair's private key to your agent
-ssh-add path_to_keypair/metin-keypair.pem
+In `azuredeploy.parameters.json` file, you will see the parameters that you can change. For example if you want to change the number of instances, you will need to change the parameter `numberOfInstances`. You can change the type of coordinator and workers separately from the parameters file. Also by default for workers, memory intense vms are used(E type) while for coordinator CPU intense vms are used(D type).
 
-# Quickly start a cluster of (1 + 2) c3.xlarge nodes at availability zone us-east-1b
-cloudformation/create-stack.sh -k metin-keypair -s FormationMetin -n 2 -i c3.xlarge -a us-east-1b
+After you run tests, you can see the results in `results` folder. The `results` folder will have the name of the config used for the test.
 
-# When your cluster is ready, it will prompt you with the connection string, connect to master node
-ssh -A ec2-user@ec2-35-153-66-69.compute-1.amazonaws.com
-```
+## <a name="azure-setup-steps"></a> Setup Steps For Each Test
+
+You will need to follow these steps to create a cluster and connect to it, on your local machine:
 
-On the coordinator node:
 ```bash
-# Setup your test cluster with PostgreSQL 10.1 and Citus master branch
-fab use.postgres:10.2 use.citus:master setup.basic_testing
 
-# Lets change some conf values 
-fab pg.set_config:max_wal_size,"'5GB'"
-fab pg.set_config:max_connections,1000
+# start ssh agent
+eval `ssh-agent -s`
 
-# And restart the cluster
-fab pg.restart
-```
+# Add your Github ssh key for enterprise (private) repo
+ssh-add
 
-On your local machine:
-```bash
-# Delete the formation
-# It's a good practice to check deletion status from the cloud formation console
-aws cloudformation delete-stack --stack-name "FormationMetin"
+# Put your public key to sshPublicKey in azuredeploy.parameters.json. You can see your public key with:
+cat ~/.ssh/id_rsa.pub
```

Comment:
can we come up with a shell command which sets the `sshPublicKey` in `azuredeploy.paramaters.json` instead of `cat`?